Gaijinizer patch/Map398.txt

```diff
@@ -4,788 +4,992 @@
-
+> CONTEXT: Map398/events/1/pages/0/372/Dialogue/enemy28
+We'll have her spend time with some other fairy
+playmates in this Fairy Realm. We have no
+intention of treating her as poorly as she
+deserves, however.#RonIffy
 > END STRING
 
 > BEGIN STRING
 悪くしないんだ！？
 
 一族の因縁なのに！？
-> CONTEXT: Map398/events/1/pages/0/377/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map398/events/1/pages/0/377/Dialogue/kazuya
+You won't do anything to her?!
 
+Even though she's a member of that family of
+beasts?!#RonIffy
 > END STRING
 
 > BEGIN STRING
 えぇ……？
 
 妖精ちゃんたちと一生遊ぶのはいいけど……
 つまり僕はずっとこのまま…？
-> CONTEXT: Map398/events/1/pages/0/382/Dialogue/enemy11 < UNTRANSLATED
-
+> CONTEXT: Map398/events/1/pages/0/382/Dialogue/enemy11
+Huh...?
+I don't mind playing with the fairies here for
+the rest of my life, but does that mean I'll be
+in here like this forever?#Ron
 > END STRING
 
 > BEGIN STRING
 いや……もっともそうな理由つけて
 結局、スフィンクスを『飼い殺し』にしたいだけだろ…
 
 魔界の噂通り、飼い殺し大好きな女王様だな……
-> CONTEXT: Map398/events/1/pages/0/387/Dialogue/other2 < UNTRANSLATED
-
+> CONTEXT: Map398/events/1/pages/0/387/Dialogue/other2
+No, she's probably just inventing a reason to
+keep you here out of the way. It looks like the
+rumors floating around the Demon Realm were true,
+this queen likes to keep a menagerie.#RonIffyWording
 > END STRING
 
 > BEGIN STRING
 ………………スフィンクスはヘラから捨てられて
 僕の具現種として過ごしてるんだ。
 
 正確には『この世界線で』ではないけど……
-> CONTEXT: Map398/events/1/pages/0/393/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map398/events/1/pages/0/393/Dialogue/kazuya
+Sphinx was abandoned by Hera, and now exists as
+one of my Spirits. Though not exactly in this
+particular "world line"...#RonIffy
 > END STRING
 
 > BEGIN STRING
 だから今の彼女は、ヘラとはいっさい関係がない。
 
 それでも、考え直してもらえないのか…？
-> CONTEXT: Map398/events/1/pages/0/398/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map398/events/1/pages/0/398/Dialogue/kazuya
+So the Sphinx here has no connection at all to
+Hera. Will you reconsider, knowing that?#Ron
 > END STRING
 
 > BEGIN STRING
 『世界線』……？
 
 ふふ……興味深いことを言うのですね、貴方は。
-> CONTEXT: Map398/events/1/pages/0/403/Dialogue/enemy28 < UNTRANSLATED
+> CONTEXT: Map398/events/1/pages/0/403/Dialogue/enemy28
+"World line"...?
 
+You've said something very interesting just now,
+fufu...#Ron
 > END STRING
 
 > BEGIN STRING
 そうですね、もちろん再考の余地はありますよ
 
 もっとも……あなたの態度次第ですがね…
-> CONTEXT: Map398/events/1/pages/0/407/Dialogue/enemy28 < UNTRANSLATED
-
+> CONTEXT: Map398/events/1/pages/0/407/Dialogue/enemy28
+Yes, of course there is room for me to reconsider
+my position. Though naturally, that does depend
+on your attitude.#Ron
 > END STRING
 
 > BEGIN STRING
 この奥の神殿遺跡………
 『統治者の間』にて、お待ちしております。
 
 話の続きは、そちらのほうでゆっくりと………
-> CONTEXT: Map398/events/1/pages/0/411/Dialogue/enemy28 < UNTRANSLATED
+> CONTEXT: Map398/events/1/pages/0/411/Dialogue/enemy28
+Further in at the temple ruins, I will be waiting
+for you in the "Space of Sovereigns".
 
+We can leisurely continue our conversation there.#Ron
 > END STRING
```

Comment:
"Space of Sovereigns" sounds like a really fancy way of saying something like "Throne Room" or something... Have any feelings on how we should phrase it?